tango_ros_common/tango_ros_native/src/tango_ros_conversions_helper.cpp

```diff
@@ -278,6 +278,10 @@ void toOccupancyGrid(const Tango3DR_ImageBuffer& image_grid,
       // the occupancy grid is using (x: right, y: up). The image is therefore
       // flipped around the x axis.
       int value = static_cast<int>(image_grid.data[j + (image_grid.height - i - 1) * image_grid.width]);
+      // Push raw data into message.
+      occupancy_grid->data.push_back(value);
+      /* Disabling this conversion for now
```

Comment:
I'm trying to find a generic solution here for all layers, the obstacle layer is playing dirty.